src/puppetlabs/trapperkeeper/services/webserver/jetty9_core.clj

```diff
@@ -526,7 +534,11 @@
   []
   (let [^ContextHandlerCollection chc (ContextHandlerCollection.)]
     {:handlers chc
-     :state (atom {:endpoints {}})
+     :state (atom {:endpoints {}
+                   :mbean-container nil
+                   :overrides-read-by-webserver false
```

Comment:
Thanks for doing this.  Feeling very ashamed now that I didn't do this in the first place.... :disappointed: